src/main/java/at/hannibal2/skyhanni/features/inventory/chocolatefactory/ChocolateFactoryBarnManager.kt

```diff
@@ -54,10 +56,13 @@ object ChocolateFactoryBarnManager {
             val amount = group("amount").formatLong()
             if (config.showDuplicateTime) {
                 val format = ChocolateFactoryAPI.timeUntilNeed(amount).format(maxUnits = 2)
-                event.chatComponent.appendText("\n§7(§a+§b$format §aof production§7)")
+                DelayedRun.runNextTick {
+                    ChatUtils.chat("§7(§a+§b$format §aof production§7)")
+                }
                 ChocolateAmount.averageChocPerSecond()
             }
             ChocolateAmount.addToAll(amount)
```

Comment:
not exactly related to your changes but this line does nothing as far as i can see